CITATION.bib

```diff
@@ -0,0 +1,12 @@
+@inproceedings {TBD,
+    author = {Aditya Sirish A Yelgundhalli and Patrick Zielinski and Reza Curtmola and Justin Cappos},
+    title = {Rethinking Trust in Forge-Based Git Security},
+    booktitle = {32nd Network and Distributed System Security Symposium (NDSSS 2025)},
```

Comment:
ndsssssss

Reply:
lessssssss sssssssss

Reply:
:snake: time to rewrite gittuf in python